src/components/RestaurantDetails/components/AdditionalInfo/AdditionalInfo.js

```diff
@@ -0,0 +1,42 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//CSS
+import './AdditionalInfo.css';
+
+function renderIcon(icon) {
+  if (icon) {
+    return <img src={icon} alt='icon' />;
+  }
+  return null;
+}
+
+function AdditionalInfo(props) {
+  const { icon, title, subtitle } = props;
+  return (
+    <div>
+      <div className='restaurantMetaInfo'>
+        {renderIcon(icon)}
+        <span>{title}</span>
+      </div>
+      <div>
+        <span className='lightWeightText'>{subtitle}</span>
+      </div>
+    </div>
+  );
+}
+
+AdditionalInfo.defaultProps = {
+  icon: null,
```

Comment:
you're expecting it to be string, so should we give null as a default value ??

Reply:
undefined since it's a string